examples/basic-mongo/config.js

```diff
@@ -0,0 +1,33 @@
+module.exports = {
+  mongoConnectionString: process.env.MONGO_CONNECTIONSTRING || 'mongodb://localhost:27017/basic-mongo-auth',
+  port: process.env.PORT || '2000',
+  session:{
+    name: 'authsessionid',
+    secret: 'thisisthesessionsecretusedtosigncookie',
+    resave: false,
+    saveUninitialized: true,
+  },
+  auth: {
```

Comment:
Moeten deze parameters ook niet configureerbaar zijn vanuit de environment van de docker-compose

Reply:
Ik zal ze configureerbaar maken, maar de default blijven zodat het voorbeeld ook gerunned kan worden zonder docker; omdat de defaults blijven ga ik ze ook niet expliciet in de docker-compose zetten; of vind je dat ook nodig?